pkg/webhook/server/server.go

```diff
@@ -282,16 +284,29 @@ func (s *Server) mutate(ctx context.Context, obj runtime.Object) (runtime.Object
 		return nil, errors.New("request is not of type apiextensions v1")
 	}
 	review.Response = s.MutationWebhook.Mutate(ctx, review.Request)
+	s.logAdmissionReview(review, "mutate")
+
 	return review, nil
 }
 
+func (s *Server) logAdmissionReview(review *admissionv1.AdmissionReview, operation string) {
+	if review.Request == nil {
+		s.log.Info(operation + " request handled nil request")
+	} else if review.Response == nil {
+		s.log.Info(fmt.Sprintf("%s request handled name %s namespace %s", operation, review.Request.Name, review.Request.Namespace))
```

Comment:
This also should be logged at debug rather than info level.
It is not expected that the response would be empty, so maybe log something like `unexpected empty response`